test/prod/RToken.test.ts

```diff
@@ -618,7 +618,7 @@ describe('RToken contract', function () {
       expect(await rToken.balanceOf(owner.address)).to.equal(amount)
       expect(await rToken.totalSupply()).to.equal(amount)
       const currentBlock = (await ethers.provider.getBlock('latest')).number
-      expect(currentBlock).to.equal(bn(mintingBlock).add(blocks))
+      expect(currentBlock).to.equal(bn(mintingBlock.toString()).add(blocks))
```

Comment:
So long as you're dealing with numbers in the JS SAFE_INT range, there's no need to cast values to strings before handing them to `bn` or `fp`.

```suggestion
      expect(currentBlock).to.equal(bn(mintingBlock).add(blocks))
```

Reply:
And the default BigNumber library places handle "BigNumberish" values as parameters very well, including JS numbers, bigints, and strings. (Just not strings in scientific notation)
```suggestion
      expect(currentBlock).to.equal(blocks.add(mintingBlock))
```